### ISSUE TITLE
Submenu Hub Widget

### ISSUE THREAD

**Issue by SaLii05** [jurialmunkey/skin.arctic.zephyr.2#212]
Hey. 

Can you test the submenu hub widget ? He doesn’t load it. 

Ty very much

**Comment by drinfernoo** [comment]
@jurialmunkey Yeah, it seems that the fallback hub widget works, but the ones set on individual items never display their contents.

**Comment by drinfernoo** [comment]
@jurialmunkey, have you seen this?

**Comment by jurialmunkey** [comment]
Works fine for me:

Submenu item widget "Recently Added"
![Honeyview_screenshot072](https://user-images.githubusercontent.com/4618719/88469519-9e0ba100-cf35-11ea-84cc-c8000fd09180.jpg)

Submenu item widget "Top Rated"
![Honeyview_screenshot073](https://user-images.githubusercontent.com/4618719/88469520-9fd56480-cf35-11ea-9005-5246cd8a1f90.jpg)

Submenu hub widget fallback "Spotlight"
![Honeyview_screenshot074](https://user-images.githubusercontent.com/4618719/88469522-a1069180-cf35-11ea-8d99-cfb22a6a96b8.jpg)
